### PR TITLE
MAINT: lib: A little bit of clean up for the new year.

### PR DESCRIPTION
**Pull request by WarrenWeckesser**

### CODE REVIEW

File: numpy/lib/_iotools.py [hunk 1]
```diff
@@ -5,17 +5,10 @@
...
-import sys
 import numpy as np
 import numpy.core.numeric as nx
 from numpy.compat import asbytes, asunicode, bytes, basestring
 
-if sys.version_info[0] >= 3:
-    from builtins import bool, int, float, complex, object, str
-    unicode = str
-else:
-    from __builtin__ import bool, int, float, complex, object, unicode, str
```
Comment (eric-wieser) [inline numpy/lib/_iotools.py]: My vague memory was that this is needed because of an `eval` in this file.

File: numpy/lib/_iotools.py [hunk 2]
```diff
@@ -65,40 +58,6 @@ def _is_bytes_like(obj):
     return True
 
 
-def _to_filehandle(fname, flag='r', return_opened=False):
```
Comment (eric-wieser) [inline numpy/lib/_iotools.py]: I wonder when the last caller of this function disappeared

File: numpy/lib/_iotools.py (hunk 1 above)
Comment (eric-wieser) [reply numpy/lib/_iotools.py]: Seems I must be thinking of another file.

**Review by eric-wieser — APPROVED** [review]
Looks reasonable, once CI passes.

**Comment by WarrenWeckesser** [comment]
Azure is having trouble installing mingw again...

**Comment by WarrenWeckesser** [comment]
Thanks Eric.

**Pull request merged**